src/lib/retire-js/RetireJsEngine.ts

```diff
@@ -221,7 +221,7 @@ export class RetireJsEngine extends AbstractRuleEngine {
 				} else if (code === 13) {
 					// If RetireJS exits with code 13, then it ran successfully, but found at least one vulnerability.
 					// Convert the output into RuleResult objects and resolve to that.
-					res(this.processOutput(stderr, invocation.rule));
+					res(this.processOutput(stdout, invocation.rule));
```

Comment:
As of [retire v3.0.2](https://github.com/RetireJS/retire.js/blob/master/node/CHANGELOG.md#302), the JSON is always written to stdout.